upb/def.h

```diff
@@ -794,6 +794,33 @@ class upb::MessageDef {
   /* Is this message a value? */
   bool value() const;
 
+  /* Is this message a doublevalue? */
```

Comment:
Can we change this to use an enum for well-known types instead?  We shouldn't need separate accessors for every kind of well-known type.

enum upb_wellknowntype {
  UPB_WELLKNOWNTYPE_NONE = 0,  /* Not a well-known type. */
  UPB_WLELKNOWNTYPE_DOUBLEVALUE = 1,
  UPB_WELLKNOWNTYPE_FLOATVALUE = 2,
  /* etc. */
}